cmd/pd-server/main.go

```diff
@@ -40,8 +41,27 @@ import (
 	"go.uber.org/zap"
 )
 
+const (
+	serviceMode    = "service"
+	tsoServiceMode = "tso-service"
+)
+
 func main() {
-	ctx, cancel, svr := createServerWrapper(os.Args[1:])
+	var (
+		ctx    context.Context
+		cancel context.CancelFunc
+		svr    bs.Server
+	)
+	if len(os.Args) >= 3 && os.Args[1] == serviceMode {
+		switch os.Args[2] {
+		case tsoServiceMode:
+			ctx, cancel, svr = tso.CreateServerWrapper(os.Args[3:])
+		default:
+			log.Fatal("service are not implemented")
```

Comment:
After log.Fatal, will the command output be friendly to user like displaying the help info and the supported service modes? If not, probably we want to log error and print help info here.